napari/benchmarks/benchmark_labels_layer.py

```diff
@@ -104,7 +109,16 @@ def time_get_value(self, n):
 
     def time_save_history(self, n):
         """Time to save history."""
-        self.layer._save_history()
+        current_slice = n // 2
+        coords = tuple([n // 2 for _ in range(2)])
```

Comment:
Sorry I'm not familiar with the benchmarks, but this feel  weirdly complex to me to what is basically `(n // 2, n // 2)`, or even `(current_slice, current_slice)` if you want to be faster.

 Would there be  another parameter that should depend on `n`, like the number of disk you create ? Or is that n dimensional disk (sphere, hypersphere) so `range(n)` ?

Reply:
Yeah, `n` is actually a parameter for image size, and in some sense a cleaner benchmark would actually just use a single coordinate independent of `n`. The strength of the new approach is it is just saving the changed pixel.

The more I look at this the more I think we should just drop the benchmark entirely. It's now benchmarking a private method that might change again in the future.

@DragaDoncila do you mind just dropping instead? Once we have a general undo/ redo functionality we can benchmark that separately

Reply:
@sofroniewn more than happy to drop. It's not really benchmarking anything meaningful at the moment.